scripts/common.py

```diff
@@ -78,6 +78,7 @@
 RELEASE_MAINTENANCE_BRANCH_REGEX = r'release-maintenance-(\d+\.\d+\.\d+)$'
 HOTFIX_BRANCH_REGEX = r'release-(\d+\.\d+\.\d+)-hotfix-[1-9]+$'
 TEST_BRANCH_REGEX = r'test-[A-Za-z0-9-]*$'
+OPEN_NEW_TAB = {'user_pref': None}
```

Comment:
I think you've got it the wrong way round. It should be: USER_PREFERENCES = {'open_new_tab_in_browser': None}.

That way, you can add more later.

Reply:
Done.